app/src/main/java/org/p2p/wallet/root/RootActivity.kt

```diff
@@ -108,7 +108,7 @@ class RootActivity :
             val findFragmentError = ClassCastException(
                 "Can't log screen open event: fragment - ${supportFragmentManager.findFragmentById(R.id.rootContainer)}"
             )
-            Timber.e(findFragmentError)
+            Timber.i(findFragmentError)
```

Comment:
no need to create entry for such error. ClassCastException always caused by onBackPressed